### DISCUSSION TITLE
Minimizing?

### DISCUSSION THREAD

**Discussion by akdor1154** [material-shell/material-shell#606] [General]
Hey all, and particularly @PapyElGringo,
Thanks for writing and maintaining this extension, I have long dreamed that TWMs could be used as a basis for a highly approachable desktop, and I'm very impressed that you've made this a reality.

I have a specific question / feedback - the one use-case I'm having trouble with is the lack of ability to minimize windows. It seems this is intentional. For my own background, I'm coming from `awesomewm` and so am used to working with a (simple) tiling window manager. I often have VSCode and a Terminal open in a split view, and I like to be able to minimize the terminal but keep it around and on the same workspace.

This is particularly useful on a small screen: on my desktop I can just shrink the terminal down to a little column and it doesn't really get in the way, but on my 14" laptop I really want to be able to reclaim the space used by the terminal.

Is there any chance you'd consider re-enabling window minimization? Or can you suggest another workflow for what I want, being: keep vscode and a terminal around together, but hide the terminal most of the time to save space.

Cheers
Jarrad

**Comment by PapyElGringo** [comment]
@akdor1154 The easiest way is to switch between maximize and split layout with the given shortcuts instead of minimizing

You could also use the integrated terminal of vscode and toggle it when needed

**Comment by kroq86** [comment]
I thing you need a vertical tiling mode, in case 1st window are minimized
